src/locales/en.json

```diff
@@ -238,13 +239,18 @@
     "basic_mask": "Multi-layer cloth mask with tight, secure fit",
     "basic_mask_short": "Cloth, thick and snug",
     "car_windows_down": "A moving car with the windows rolled down",
+    "car_windows_down_breakdown": "for car",
     "close_distance": "Close (less than 1 foot / 0.3 meters apart)",
+    "close_distance_breakdown": "for close distance",
     "filtered": "Indoors with a HEPA filter (flow rate 5x room size per hour)",
+    "filtered_breakdown": "for filtered air",
     "filtered_mask": "Cloth mask with removable filter insert",
     "filtered_mask_short": "Filter Insert",
     "indoor": "Indoor",
     "intimate_distance": "Kissing",
+    "intimate_distance_breakdown": "for kissing",
     "loud_voice": "Loud talking (shouting, talking over music, singing)",
+    "loud_voice_breakdown": "for loud",
```

Comment:
for increased particles generated by loud talking.

Reply:
again really long